io/read_all_test.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+import { assert, assertEquals } from "../assert/mod.ts";
+import { readAll, readAllSync } from "./read_all.ts";
+import { Buffer } from "./buffer.ts";
+import { init } from "./_test_common.ts";
+
+Deno.test("readAll()", async () => {
+  const testBytes = init();
+  assert(testBytes);
+  const reader = new Buffer(testBytes.buffer);
```

Comment:
We rely on `Buffer()` here but should work on removing that dependency in the future (in a PR).